### PR TITLE
Foursquare Badge Support

### PR DESCRIPTION
**Pull request by bassdread**
Adds badge fetching and storing to Foursquare.

### CODE REVIEW

File: Connectors/foursquare/foursquare.synclet [hunk 1]
```diff
@@ -13,7 +13,12 @@
         {
             "frequency": 60,
             "name": "recent"
+        },
+        {
+            "frequency": 5,
```
Comment (amcjen) [inline Connectors/foursquare/foursquare.synclet]: Would you mind kicking this frequency up to something like 3600.  Probably around the same amount of scheduling frequency as new contacts.

**Comment by amcjen** [comment]
Woo!  Looks like you got it working!  Made one comment on the frequency. Otherwise looks great.

**Comment by bassdread** [comment]
No problem, I left the debug value in by mistake. I bumped it to 3600.

**Comment by amcjen** [comment]
Awwwesome!

**Pull request merged**